django_app/redbox_app/settings.py

```diff
@@ -147,11 +147,12 @@
 CSP_DEFAULT_SRC = (
     "'self'",
     "s3.amazonaws.com",
+    "plausible.io",
```

Comment:
Do we need this in DEFAULT as well as SCRIPT?

Reply:
Yes just tried with out and it thew an CSP error